sips/pending/_posts/2017-01-13-binary-compatibility.md

```diff
@@ -0,0 +1,168 @@
+---
+layout: sip
+title: SIP XX - Improving binary compatibility with @binaryCompatible
+disqus: true
+---
+
+__Dmitry Petrashko__
+
+__first submitted 13 January 2017__
+
+## Introduction ##
```

Comment:
I think that the introduction would benefit from the definitions of the following terms used later in the document:
1) Binary compatibility (both the intransitive and transitive variations used below)
2) Public API
3) Binary signature
4) The notion of signatures being different from those written by their authors.

I'd suggest having a small example that introduces and illustrates these notions.